src/cli/bootupctl.rs

```diff
@@ -130,26 +130,14 @@ impl CtlCommand {
     /// Runner for `update` verb.
     fn run_update() -> Result<()> {
         ensure_running_in_systemd()?;
-        let sysroot = openat::Dir::open("/").context("Opening root dir")?;
-        let dest_fd = format!("/proc/self/fd/{}", sysroot.as_raw_fd());
-        let dest_root = std::fs::read_link(dest_fd)?;
```

Comment:
Hmm so actually we always operated on `/`, and this code pattern was just a really complicated way to go from `/` -> file descriptor -> `/` effectively.

The only tricky thing here is that eventually we actually do want to operate on the non-booted root! See https://github.com/coreos/bootupd/issues/108